tests/validator_test.go

```diff
@@ -196,6 +196,7 @@ var _ = Describe("Template validator operand", func() {
 	})
 
 	It("[test_id:4913] should successfully start template-validator pod", func() {
+		strategy.SkipUnlessHighlyAvailableTopologyMode()
```

Comment:
This line is a little confusing to me. Would it be better to call the function `SkipForSingleReplicaTopologyMode()` ?

Reading this, I need to know that HighlyAvailableTopology == not SingleReplicaTopology, that there are no other choices.